### PR TITLE
[KYUUBI #1011] Fallback some frontend service config option about thrift

### PR DESCRIPTION
**Pull request by yanghua**
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala [hunk 1]
```diff
@@ -235,61 +235,118 @@ object KyuubiConf {
...
+  val FRONTEND_THRIFT_BIND_PORT: ConfigEntry[Int] = buildConf("frontend.thrift.bind.port")
+    .doc("Port of the machine on which to run the thrift frontend service.")
+    .version("1.4.0")
+    .fallbackConf(FRONTEND_BIND_PORT)
+
+  @deprecated(s"using ${FRONTEND_THRIFT_MIN_WORKER_THREADS.key} instead", "1.4.0")
   val FRONTEND_MIN_WORKER_THREADS: ConfigEntry[Int] = buildConf("frontend.min.worker.threads")
-    .doc("Minimum number of threads in the of frontend worker thread pool for the frontend" +
-      " service")
+    .doc("Minimum number of threads in the of frontend worker thread pool for the thrift " +
```
Comment (cfmcgrady) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala:266]: Shall we also add the keyword `deprecated` for the `doc`?

File: kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala [hunk 2]
```diff
@@ -235,61 +235,118 @@ object KyuubiConf {
...
+      "frontend service")
+    .version("1.4.0")
+    .fallbackConf(FRONTEND_MIN_WORKER_THREADS)
+
+  @deprecated(s"using ${FRONTEND_THRIFT_MAX_WORKER_THREADS.key} instead", "1.4.0")
   val FRONTEND_MAX_WORKER_THREADS: ConfigEntry[Int] = buildConf("frontend.max.worker.threads")
-    .doc("Maximum number of threads in the of frontend worker thread pool for the frontend" +
-      " service")
+    .doc("Maximum number of threads in the of frontend worker thread pool for the thrift " +
```
Comment (cfmcgrady) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala:281]: ditto.

**Comment by yanghua** [comment]
cc @yaooqinn @pan3793

**Review by cfmcgrady — APPROVED** [review]

**Review by yaooqinn — APPROVED** [review]

**Review by ulysses-you — APPROVED** [review]

**Comment by ulysses-you** [comment]
@yanghua thank you, merging to master for 1.4.0

**Pull request closed without merging**